test/api_test.exs

```diff
@@ -86,6 +86,14 @@ defmodule Surface.APITest do
     message = ~r/cannot use name "label". There's already a prop/
     assert_raise(CompileError, message, fn -> eval(code) end)
 
+    code = """
+    prop avatar, :string
+    upload avatar, accept: :any
+    """
+
+    message = ~r/cannot use name "avatar". There's already a prop/
```

Comment:
Does not make sense... This is a remnant of the implementation as a `data`. I'll remove it!